docs/getting_started/requirements.md

```diff
@@ -94,8 +96,8 @@ Optionally if you intend to edit [PNG, SVG, GIF or WEBP files in the Image Edito
 
 ## Asset manager
 
-- `Node.js` 14
-- `yarn` 1.15.2 or higher
+- `Node.js` 14+
```

Comment:
I think the plus sign next to Node could be tricky. 
People could interpret it to mean 16, 18 etc.
Summon @dew326

Reply:
@micszo We have a big note at the top of the page that says: If you see a “+” next to the product version, it means that we recommend this version or higher within the same major release. For example, “Nginx 1.18+” means any 1.x version higher or equal to 1.18, but not 2.x.

Reply:
Yeah, I've seen it. 😉 But I would be calmer if someone from JS team confirmed it's clear enough for Node.

Reply:
@dew326 even so, we cannot remove node 14 for v4 for legal reasons, as @alongosz explained to me

Reply:
It would be really great to have a dependencies page where you select your product and then it lists all the scheduled deprecations and EOLs we know about, in an easy to read overview. Something like https://www.php.net/supported-versions.php only covering more deps. Would require some development :)

For now, sprinkling notes here and there is much better than nothing. +1 to this. And while we're at it, we should also note that PHP 7.4 is EOL, _unless_ you have extended support from vendors like Debian or Zend.

Reply:
Let's go with additional notes here and there as we are mid v4, we will think about redesigning and better showcasing the dependencies for LTS or v5.

Reply:
For Node.js, I would add a note that it is EOL after 2023-04-30. I don't want Requirements page turning into a calendar with dates that we need to keep track of and update.